src/app/core/hal/feeder.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HalBaseService } from 'ngx-prx-styleguide';
+import { Env } from '../core.env';
+
+@Injectable()
+export class FeederService extends HalBaseService {
+
+  get host(): string {
+    return 'feeder.prx.org'; // TODO
```

Comment:
We don't actually use FeederService yet, so left this as a TODO.  (Currently to get to feeder, we just navigate via an absolute-link found in a CMS series/story).